package.json

```diff
@@ -0,0 +1,340 @@
+    "range-parser": "^1.2.1",
+    "randombytes": "^2.1.0",
+    "readable-stream": "^3.6.0",
+    "raw-body": "^2.5.1",
+    "readdirp": "^3.6.0",
+    "rechoir": "^0.7.1",
+    "regenerate": "^1.4.2",
+    "regenerator-runtime": "^0.13.9",
+    "regenerate-unicode-properties": "^10.0.1",
+    "regenerator-transform": "^0.15.0",
+    "regexpu-core": "^5.1.0",
+    "regjsgen": "^0.6.0",
+    "regjsparser": "^0.8.4",
+    "relateurl": "^0.2.7",
+    "renderkid": "^3.0.0",
+    "require-from-string": "^2.0.2",
+    "requires-port": "^1.0.0",
+    "resolve-cwd": "^3.0.0",
+    "resolve": "^1.22.1",
+    "resolve-from": "^5.0.0",
+    "rimraf": "^3.0.2",
+    "retry": "^0.13.1",
+    "safe-buffer": "^5.2.1",
+    "safer-buffer": "^2.1.2",
+    "select-hose": "^2.0.0",
+    "selfsigned": "^2.0.1",
+    "semver": "^7.3.7",
+    "send": "^0.18.0",
+    "serialize-javascript": "^6.0.0",
+    "serve-index": "^1.9.1",
+    "setprototypeof": "^1.2.0",
+    "serve-static": "^1.15.0",
+    "shallow-clone": "^3.0.1",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "schema-utils": "^3.1.1",
+    "side-channel": "^1.0.4",
+    "sockjs": "^0.3.24",
+    "signal-exit": "^3.0.7",
+    "source-map": "^0.6.1",
+    "source-map-js": "^1.0.2",
+    "source-map-support": "^0.5.21",
+    "spdy": "^4.0.2",
+    "spdy-transport": "^3.0.0",
+    "statuses": "^2.0.1",
+    "string_decoder": "^1.3.0",
+    "strip-ansi": "^6.0.1",
+    "strip-final-newline": "^2.0.0",
+    "style-loader": "^3.3.1",
+    "supports-color": "^8.1.1",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "tapable": "^2.2.1",
+    "terser-webpack-plugin": "^5.3.6",
+    "thunky": "^1.1.0",
+    "to-fast-properties": "^2.0.0",
+    "terser": "^5.15.0",
+    "toidentifier": "^1.0.1",
+    "to-regex-range": "^5.0.1",
+    "tslib": "^2.4.0",
+    "type-is": "^1.6.18",
+    "unicode-canonical-property-names-ecmascript": "^2.0.0",
+    "unicode-match-property-ecmascript": "^2.0.0",
+    "unicode-match-property-value-ecmascript": "^2.0.0",
+    "unicode-property-aliases-ecmascript": "^2.0.0",
+    "update-browserslist-db": "^1.0.5",
+    "unpipe": "^1.0.0",
+    "uri-js": "^4.4.1",
+    "utila": "^0.4.0",
+    "util-deprecate": "^1.0.2",
+    "util": "^0.10.4",
+    "utils-merge": "^1.0.1",
+    "uuid": "^8.3.2",
+    "vary": "^1.1.2",
+    "watchpack": "^2.4.0",
+    "wbuf": "^1.7.3",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-middleware": "^5.3.3",
+    "webpack-dev-server": "^4.10.1",
+    "webpack-merge": "^5.8.0",
+    "websocket-driver": "^0.7.4",
+    "webpack-sources": "^3.2.3",
+    "websocket-extensions": "^0.1.4",
+    "which": "^2.0.2",
+    "wrappy": "^1.0.2",
+    "wildcard": "^2.0.0",
+    "ws": "^8.8.1",
+    "yallist": "^4.0.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "webpack --config webpack.config.js",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production"
+  },
```

Comment:
- [ ]  Like the previous reviewers noted, you are missing scripts like start, which are needed to run your app. You can check the examples below to get an idea of how to write "start" scripts. There should be a way to start the project, say `npm run start`

**EX:**
```
"scripts": {
    "start": "webpack --config webpack.config.js --watch"
  }
  
  ```
  Or,
```
  "scripts": {
     "start": "webpack serve --open"
  }
```